subworkflows/local/alevin.nf

```diff
@@ -66,6 +64,6 @@ workflow SCRNASEQ_ALEVIN {
 
     emit:
     ch_versions
-    alevin_results = SIMPLEAF_QUANT.out.alevin_results
-    alevinqc = ALEVINQC.out.report
+    alevin_results = SIMPLEAF_QUANT.out.alevin_results.map{ meta, files -> [meta + [input_type: 'raw'], files] }
```

Comment:
Alevin should also provided filtered and unfiltered results in one way or another, I think. But maybe this is also addressed by #361

Reply:
Let's have this addressed in #361.